examples/webgl_framebuffer_texture.html

```diff
@@ -103,7 +103,8 @@
 
 				//
 
-				texture = new THREE.FramebufferTexture( textureSize, textureSize );
+				texture = new THREE.DataTexture( new Uint8Array( textureSize * textureSize * 4 ), textureSize, textureSize );
+				texture.needsUpdate = true;
```

Comment:
Creating an "empty" texture that is only useful for copying data into, as FramebufferTexture was, is a bit more awkward, now. In the past I've used RenderTargets to serve as "empty" texture vessels for copying data around into but there's still some extra memory overhead in that case (depth buffers, etc).